src/app/clusters/test-cluster-server/test-cluster-server.cpp

```diff
@@ -44,6 +44,8 @@ using namespace chip::app::Clusters::TestCluster::Attributes;
 constexpr const char * kErrorStr = "Test Cluster: List Octet cluster (0x%02x) Error setting '%s' attribute: 0x%02x";
 #endif // CHIP_CLUSTER_CONFIG_ENABLE_COMPLEX_ATTRIBUTE_READ
 
+constexpr uint8_t kAttributeCount = 4;
```

Comment:
It took me a while to figure out what this, mostly because I think this isn't quite the right name. This tracks the maximum number of list items right? If so, can we name it so?